### DISCUSSION TITLE
Notification when a client connects / disconnects

### DISCUSSION THREAD

**Discussion by Refragg** [LizardByte/community#197] [Sunshine]
Hello, i was trying this thing out and it works great, i was wondering if there is anyway we can have some notification if someone joins or quits an application? I found out that you can use the command preparations but this only works when the app is started and stopped which is not exactly it and, there's no easy way to get information on the client that connected, sure, you can put the logs to verbose to have it spit out the IP and some other stuff but i feel like it could be nice to have directly inside Sunshine.

**Comment by ReenigneArcher** [comment]
I agree, webhooks or something would be nice when certain events occur.

**Comment by ReenigneArcher** [comment]
There are system tray notifications in Sunshine now. I guess you are wanting something beyond that though?

**Reply by Refragg** [reply]
Hi, to be fair i cannot remember the specifics of the use case i was planning with this. However, this definitely is a step in the right direction! Although from my little amount of testing, it looks like there's still no clear way to identify the client that connected, maybe there could in the future be an option somewhere to log some information about the client that just connected? Great progress nonetheless :+1:.

**Reply by TCAtrevor** [reply]
The notification only shows when a stream starts, but not if multiple people connect afterwards. I would really like if it showed one every time a new connection was made.

**Reply by gnattu** [reply]
I don’t like this notification and I don’t want to see a popup and hear a sound each and every time I connect, can I disable that?

**Reply by ReenigneArcher** [reply]
In Windows 11 you can disable notifications for a specific application. Have you tried that?

**Reply by gnattu** [reply]
Yes, that works

**Reply by LuminarySage** [reply]
I'm not getting any notifications when a client connects and I don't see sunshine anywhere in windows notification settings. What am I doing wrong?
